quickstart/giphy/pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires-python = ">=3.11"
 license = "Apache-2.0"
 readme = "README.md"
 dependencies = [
-    "atlan-application-sdk[workflows,tests]==0.1.1rc23",
+    "atlan-application-sdk[mcp,pandas,workflows]==0.1.1rc23",
```

Comment:
why do we need pandas?

Reply:
also the version bump depends on merging of the SDK PR https://github.com/atlanhq/atlan-sample-apps/pull/50